app/home.html

```diff
@@ -1,42 +0,0 @@
-<!DOCTYPE html>
```

Comment:
The like and share buttons aren't necessary for the project really. We could move them to the navbar if we really want? Idk what we want to do with that.